Clarify Block and inline boxes (#23748)
mdn/content: files/en-us/learn/css/building_blocks/the_box_model/index.md

## Block and inline boxes

> In CSS we have several types of boxes. The two most common are **block boxes** and **inline boxes**. The type refers to how the box behaves in terms of page flow and in relation to other boxes on the page. Boxes have an **inner display type** and an **outer display type**.

Reviewer: ```suggestion
In CSS we have several types of boxes that generally fit into the categories of **block boxes** and **inline boxes**. The type refers to how the box behaves in terms of page flow and in relation to other boxes on the page. Boxes have an **inner display type** and an **outer display type**.
```

Author: Yes this is better.
